docs/sources/operations/multi-tenancy.md

```diff
@@ -17,3 +17,19 @@ To run in multi-tenant mode, Loki should be started with `auth_enabled: true`.
 Loki can be run in "single-tenant" mode where the `X-Scope-OrgID` header is not
 required. In single-tenant mode, the tenant ID defaults to `fake`.
 
+## Multi-tenant Queries
```

Comment:
I wonder whether we should document how to setup a datasource in Grafana for multi-tenant queries. It is not trivial IMHO.